app/validators/bill-runs/setup/region.validator.js

```diff
@@ -10,7 +10,8 @@ const Joi = require('joi')
 /**
  * Validates data submitted for the `/bill-runs/setup/{sessionId}/region` page
  *
- * @param {object} payload - The payload from the request to be validated
+ * @param {object} data - The payload from the request to be validated
```

Comment:
Should these be updated to say 'The data' etc. 

i see all others are the same so resolve if a none issue.

Reply:
Fair comment. TL;DR; meh 🤷 

Regarding all the validators, it's just data that needs to be validated. Occasionally, a validator checks something that _didn't_ come via a payload.

So we were going for something 'generic'. But, in most cases, it is coming from a payload, and there are other JSDoc comments where we call out where we expect the param to be coming from, for example, the result of a fetch service. Hence, the mention in the comment.

I'm happy for others to wade in before merging.

Reply:
Cool. 

I am not precious. 

I will resolve can always adjust in the future.